### PR TITLE
Exercise info screen

### PR DESCRIPTION
**Pull request by Gotalicp**
The ui for the exercise info screen
Instead of using 3 different screens is now 1

Old: 

New:

### CODE REVIEW

File: app/src/main/java/bg/zahov/app/ui/custom/CommonLineChart.kt [hunk 1]
```diff
@@ -25,7 +24,7 @@ fun CommonLineChart(
     modifier: Modifier = Modifier,
     data: LineChartData
 ) {
-    val color =  MaterialTheme.colorScheme.secondary.toArgb()
+    val color = MaterialTheme.colorScheme.secondary.toArgb()
```
Comment (dimitarstoyanovtumba) [inline app/src/main/java/bg/zahov/app/ui/custom/CommonLineChart.kt:27]: nit: name can be more specific

File: app/src/main/java/bg/zahov/app/ui/exercise/info/ExerciseInfoScreen.kt [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+import bg.zahov.app.data.model.MeasurementType
+import bg.zahov.app.ui.custom.CommonLineChart
+import bg.zahov.app.ui.theme.FitnessTheme
+import bg.zahov.fitness.app.R
+import com.github.mikephil.charting.data.Entry
+
+
+@Composable
+fun ExerciseInfoScreen() {
+    val testHistory = listOf(
```
Comment (dimitarstoyanovtumba) [inline app/src/main/java/bg/zahov/app/ui/exercise/info/ExerciseInfoScreen.kt:53]: nit: test data can be in a separate file for easier reading

File: app/src/main/java/bg/zahov/app/ui/exercise/info/ExerciseInfoScreen.kt [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+            fontWeight = FontWeight.Bold,
+            style = MaterialTheme.typography.titleMedium
+        )
+
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceAround
+        ) {
+            Column {
+                Text(
```
Comment (dimitarstoyanovtumba) [inline app/src/main/java/bg/zahov/app/ui/exercise/info/ExerciseInfoScreen.kt:295]: Those texts are in `Column { Row { Column { Text() } } }` structure. Perhaps it's a good idea to use maxLines and ellipsize for them?

File: app/src/main/java/bg/zahov/app/ui/custom/CommonLineChart.kt (hunk 1 above)
Comment (Gotalicp) [reply app/src/main/java/bg/zahov/app/ui/custom/CommonLineChart.kt]: fixed

File: app/src/main/java/bg/zahov/app/ui/exercise/info/ExerciseInfoScreen.kt (hunk 2 above)
Comment (Gotalicp) [reply app/src/main/java/bg/zahov/app/ui/exercise/info/ExerciseInfoScreen.kt]: 👍

File: app/src/main/java/bg/zahov/app/ui/exercise/info/ExerciseInfoScreen.kt (hunk 3 above)
Comment (Gotalicp) [reply app/src/main/java/bg/zahov/app/ui/exercise/info/ExerciseInfoScreen.kt:281]: Its is column { Row { Column {text, text} Column {text, text}}. I wanted to have two columns with two textfield which are on both ends of the row. I am not really sure how to do it another way

Comment (dimitarstoyanovtumba) [reply app/src/main/java/bg/zahov/app/ui/exercise/info/ExerciseInfoScreen.kt:305]: I see no issue in the arrangement. I just pointed out that it indicates multiple texts on the same row. 
The question is if you think those texts should be elipsized and constraint in any way. Did you test this screen with large text system setting?

**Pull request merged**